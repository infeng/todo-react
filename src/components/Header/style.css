header h1 {
  color: rgba(175, 47, 47, 0.15);
  font-size: 6rem;
}
header {
  margin-bottom: 20px;
}