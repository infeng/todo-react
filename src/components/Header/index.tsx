import * as React from 'react';
import './style.css';

export default class Header extends React.Component<any, any> {
  render() {
    return (
      <header>
        <h1>Todos</h1>
      </header>
    );
  }
}
