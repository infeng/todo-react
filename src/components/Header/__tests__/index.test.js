import React from 'react';
import { mount, render } from 'enzyme';
import Header from '..';

function createHeader() {
  return (
    <Header/>
  );
}

describe('Header', () => {
  it('render Header correctly', () => {
    let header = createHeader();

    const wrapper = render(header);
    expect(wrapper).toMatchSnapshot();

    const wrapper2 = mount(header);
    expect(wrapper2.text()).toBe('Todos');
  });
});