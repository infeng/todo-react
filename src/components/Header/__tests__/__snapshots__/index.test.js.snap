// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render Header correctly 1`] = `
<header>
  <h1>
    Todos
  </h1>
</header>
`;
